code/Test_definitions/carrier-billing-cancelPayment.feature

```diff
@@ -0,0 +1,184 @@
+Feature: CAMARA Carrier Billing API, v0.3 - Operation cancelPayment
+  # Input to be provided by the implementation to the tester
+  #
+  # Implementation indications:
+  # * Endpoint required for 2-step payment
+  #
+  # Testing assets:
+  # * A phone number eligible for payment (no restrictions for it to be used to perform a payment)
+  #
+  # References to OAS spec schemas refer to schemas specifies in carrier-billing.yaml, version 0.3.0
+
+  Background: Common cancelPayment setup
+    Given the resource "/carrier-billing/v0.3/payments/{paymentId}/cancel"                                                              |
+    And the header "Content-Type" is set to "application/json"
+    And the header "Authorization" is set to a valid access token
+    And the header "x-correlator" is set to a UUID value
+    And the path parameter "paymentId" is set to a valid value
+    And the request body is set by default to a request body compliant with the schema
+
+  ##############################
+  # Happy path scenarios
+  ##############################
+
+
+  @cancel_payment_01_generic_success_scenario
+  Scenario: Common validations for any success scenario
+    # Valid default request body compliant with the schema
```

Comment:
`given` clause is not missing?
I was expecting to have `Given the resource "/carrier-billing/v0.3/payments/{paymentId}/cancel"   `

Reply:
Yes, I will check given for cancel and confirm. But probably using

"Given the request body is set to a valid request body"

As `Given the resource "/carrier-billing/v0.3/payments/{paymentId}/cancel"` is used in Background